.github/workflows/package_for_test.yml

```diff
@@ -94,43 +94,53 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Upload Neuron App Zip
+<<<<<<< HEAD
+        if: matrix.os == 'macos-latest'
+        uses: actions/upload-artifact@v3
+=======
         if: matrix.os == 'macos-11'
         uses: actions/upload-artifact@v2
+>>>>>>> 4fc59fc47140cfbdc0b3cdf4088f50d7842fa86e
         with:
           name: Neuron-Mac
           path: release/Neuron-*-mac-x64.zip
 
       - name: Upload Neuron App Zip(arm64)
         if: matrix.os == 'macos-latest'
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v3
         with:
           name: Neuron-Mac
           path: release/Neuron-*-mac-arm64.zip
 
       - name: Upload Neuron Dmg
+<<<<<<< HEAD
+        if: matrix.os == 'macos-latest'
+        uses: actions/upload-artifact@v3
+=======
         if: matrix.os == 'macos-11'
         uses: actions/upload-artifact@v2
+>>>>>>> 4fc59fc47140cfbdc0b3cdf4088f50d7842fa86e
```

Comment:
Conflicts here.

Reply:
Conflicts were resolved by https://github.com/nervosnetwork/neuron/pull/2514/commits/42b346305a1e93992679ee3eb4196f008b6a6e6d. I also formated the yaml files because double quotes and single quotes were both used.